app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -15,16 +15,18 @@ CROS_WORKON_LOCALNAME="rkt"
 CROS_WORKON_REPO="git://github.com"
 
 if [[ "${PV}" == "9999" ]]; then
-	KEYWORDS="~amd64"
+	KEYWORDS="~amd64 ~arm64"
 else
-	KEYWORDS="amd64"
-	CROS_WORKON_COMMIT="886c62d84158b1164a91bef9b4ca68de4c4e0322" # v1.9.1
+	KEYWORDS="amd64 arm64"
+	CROS_WORKON_COMMIT="cb11bdfa3fa85239d321b937cdb1d0d9b834fbd6" # v1.10.0
 fi
 
+COREOS_BOARD="${ARCH}-usr"
+
 PXE_VERSION="1068.0.0"
 PXE_SYSTEMD_VERSION="v229"
-PXE_URI="https://alpha.release.core-os.net/amd64-usr/${PXE_VERSION}/coreos_production_pxe_image.cpio.gz"
-PXE_FILE="${PN}-pxe-${PXE_VERSION}.img"
+PXE_URI="https://alpha.release.core-os.net/${COREOS_BOARD}/${PXE_VERSION}/coreos_production_pxe_image.cpio.gz"
+PXE_FILE="${PN}-pxe-${COREOS_BOARD}-${PXE_VERSION}.img"
 
 SRC_URI="rkt_stage1_coreos? ( $PXE_URI -> $PXE_FILE )"
```

Comment:
We could do something like this, but it is just guessing at how the mirror selects files:

```
PXE_URI_AMD64="https://alpha.release.core-os.net/amd64-usr/${PXE_VERSION}/coreos_production_pxe_image.cpio.gz"
PXE_URI_ARM64="https://alpha.release.core-os.net/arm64-usr/${PXE_VERSION}/coreos_production_pxe_image.cpio.gz"

PXE_FILE_AMD64="${PN}-pxe-amd64-usr-${PXE_VERSION}.img"
PXE_FILE_ARM64="${PN}-pxe-arm64-usr-${PXE_VERSION}.img"

SRC_URI="rkt_stage1_coreos? (
	amd64? ( ${PXE_URI_AMD64} -> ${PXE_FILE_AMD64} )
	arm64? ( ${PXE_URI_ARM64} -> ${PXE_FILE_ARM64} )
)"
```
@marineam What do you think?